src/@providers/README.md

```diff
@@ -1,9 +1,9 @@
-# cldctl providers
+# Architect Providers
 
-Providers are arcctl plugins that can list, get, create, update, and/or delete the [resources](../%40resources/) supported by arcctl. Developers can register their cloud provider "accounts" with
-arcctl in order to let the provider help them create cloud [resources](../%40resources/) more easily.
+Providers are Architect's plugins that can list, get, create, update, and/or delete the [resources](../%40resources/) supported by Architect. Developers can register their cloud provider "accounts" with
```

Comment:
This sentence feels odd. `Providers are Architect's plugins` maybe we do not need the word `Architect` here.